pkgs/by-name/ku/kustomize-lint/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  kustomize-lint,
+  testers,
+}:
+
+buildGoModule rec {
+  pname = "kustomize-lint";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "groq";
+    repo = "kustomize-lint";
+    rev = "v${version}";
```

`tag` is preferred [Nixpkgs Reference](https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub)

```suggestion
    tag = "v${version}";
```